grails-app/views/stockMovement/_summary.gsp

```diff
@@ -91,37 +92,60 @@
                         <g:if test="${!stockMovement?.dateRequested }">
                             <span class="dateRequested">
                                 <warehouse:message code="stockMovement.dateRequested.label"/>:
-                                <label><format:date obj="${stockMovement?.dateRequested}"/></label>
+                                <label>
+                                    <g:formatDate
+                                            formatName="${DateFormatName.ABBREVIATED_MONTH.property}"
+                                            date="${stockMovement?.dateRequested}"
```

Comment:
I need more time to think about this and review what's currently happening, but I'm going to post the rest of my review as-is.